spring-integration-core/src/main/java/org/springframework/integration/context/IntegrationContextUtils.java

```diff
@@ -131,12 +134,23 @@ public static TaskScheduler getRequiredTaskScheduler(BeanFactory beanFactory) {
 
 	/**
 	 * @param beanFactory BeanFactory for lookup, must not be null.
-	 * @return the instance of {@link StandardEvaluationContext} bean whose name is "integrationEvaluationContext" .
+	 * @return the instance of {@link StandardEvaluationContext} bean whose name is
+	 * {@value #INTEGRATION_SIMPLE_EVALUATION_CONTEXT_BEAN_NAME}.
```

Comment:
Must be `INTEGRATION_EVALUATION_CONTEXT_BEAN_NAME`.

Will fix on merge if only concern.

Reply:
Oops; good catch.